lib/rubocop/cop/layout/empty_line_between_defs.rb

```diff
@@ -76,28 +76,40 @@ module Layout
       #   # good
       #   class ErrorA < BaseError; end
       #   class ErrorB < BaseError; end
-      #   class ErrorC < BaseError; end
       #
       #   # good
       #   class ErrorA < BaseError; end
       #
       #   class ErrorB < BaseError; end
       #
-      #   class ErrorC < BaseError; end
+      #   # good
+      #   def foo = 123
+      #   def bar = 456
+      #
+      #   # good
+      #   def foo = 123
+      #
+      #   def foo = 456
```

Comment:
Can you describe examples with macro instead of an endless method definition, in line with the intent of this PR?